rust-toolchain.toml

```diff
@@ -1,3 +1,3 @@
 [toolchain]
-channel = "1.65.0"
+channel = "1.75.0"
```

Comment:
We can't update the Rust toolchain without also updating zcashd, this will need to be reverted.

Reply:
that sounds like a quest, my good fellow!

Reply:
OK I think I can change the async functions that @str4d used in the FsBlockDb implementation to sync versions. I will just return a join handle for delete scanned. Damn I wish I knew this sooner... I was wondering why the tool chain was fixed!